components/team_controller/actions/index.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
Actions defined in this file are only used in TeamController, hence moved it inside of the same folder instead of "actions/xxx"